api_gateway/src/main/kotlin/pl/edu/pg/rsww/apigateway/RoutingController.kt

```diff
@@ -27,6 +25,36 @@ public class RoutingController(
         return "Hello, I'm alive!"
     }
 
+    @RequestMapping("/api/mocked/{*path}")
+    fun mock(
+        @PathVariable path: String,
+        @RequestParam params: Map<String, String>,
+    ): ResponseEntity<String> {
+        val builder = ResponseEntity.status(200)
+        if (path.contains("get_trips")) {
```

Comment:
(Technically not important for the frontend mockup so feel free to ignore)

Assuming that this will be based on cursors (or equivalent pointers) in the future, it would make sense for the request to require an `after` param with an ID/cursor after which the next result should be (so in the case of the mock id of the returned trip would be `after + 1`) and such cursor would be returned with the response under the `next` key (so in the case of the mock it would be equivalent to `id`, in the final implementation it would depend on the used DB queries).